#Config file for calculating 3D position of Object in bounding box

depth_topic: "/camera/depth/points"
boundingbox_sub: "/darknet_ros/bounding_boxes"
result_topic: "/detection/center_depth"
tf_frame_source: "camera_depth_link"
tf_frame_target: "map"
map_file_path: "/../data/map_ycb_areas.json"
service_get_current_area_name: "get_current_area_in_map"
service_get_objects_in_room: "get_objects_list"
